src/main/java/com/eprosima/fastcdr/idl/templates/TypesSource.stg

```diff
@@ -1020,7 +908,7 @@ bool b = true;
 
 switch(m__d)
 {
-    $totallabels:{case $it$:}; separator="\n"$
+    $totallabels:{ label | case $label$:}; separator="\n"$
     b = false;
     break;
     default:
```

Comment:
```suggestion
        b = false;
        break;
    default:
```

Also the break below the default case and also after the `$else$`.

Reply:
Done in 3d798c2